share/spice/yoga_asanas/yoga_asanas.js

```diff
@@ -0,0 +1,46 @@
+
+function ddg_spice_yoga_asanas(apiResult) {
+
+    if (!apiResult || !apiResult.response.numFound) {
+        return Spice.failed('yoga_asanas');
+    }
+
+    var docs = apiResult.response.docs,
+        first = JSON.parse(docs[0].meta).order,
+        last = JSON.parse(docs[docs.length - 1].meta).order,
+        asanas = {
+            id: 'yoga_asanas',
+            name: 'Yoga Asanas',
+            data: docs,
+            normalize: function(item){
+                var meta = JSON.parse(item.meta);
+                // if not deleted will break "More at" links
+                delete item.meta;
+
+                return {
+                    image: meta.img,
+                    altSubtitle: item.paragraph,
+                    url: meta.srcUrl,
+                    srcName: meta.srcName,
+                    srcUrl: meta.srcUrl,
+                    srcIcon: meta.favicon
```

Comment:
Possible for any of these to not exist? Might make the results look weird if any one of these is.